core/test/unit/server_helpers/ghost_head_spec.js

```diff
@@ -42,7 +42,7 @@ describe('{{ghost_head}} helper', function () {
                 return localSettingsCache[key];
             });
 
-            configUtils.set('url', 'http://localhost:82832/');
+            configUtils.set('url', 'http://localhost:65530/');
```

Comment:
I had to change the port for testing here, as `got` only works with ports < `65539` and we're not stubbing the `getImageSize` fn yet.